src/test/java/nl/altindag/log/LogCaptorShould.java

```diff
@@ -382,6 +386,45 @@ void captureMdcHeadersWhereSlf4jIsUsed() {
         assertMdcHeaders(logCaptor, "test-slf4j-mdc", "hello-slf4j");
     }
 
+    @Test
+    void detachWithTearDown() {
```

Comment:
Maybe remain this method as we dont have the teardown method anymore? Something like `detachAppenderWithCloseMethod`

Reply:
Done